### PR TITLE
=act #20982 increase timeout in CircuitBreakerSpec

### LINKED ISSUE #20982: FAILED: CircuitBreakerSpec
Timeout was too tight (see the GC message):

```
[info] CircuitBreakerSpec:
2215.391: [GC concurrent-cleanup-end, 0.0001002 secs] <<<<<<<<
[info] - must increment failure count on callTimeout before call finishes *** FAILED *** (152 milliseconds)
[info]   java.lang.AssertionError: assertion failed: timeout 100 milliseconds expired: <<<<<<
[info]   at scala.Predef$.assert(Predef.scala:170)
[info]   at akka.testkit.TestKitBase$class.poll$1(TestKit.scala:239)
```

### PR DESCRIPTION
**Pull request by ktoso**
Resolves #20982 CircuitBreakerSpec

### CODE REVIEW

File: akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala [hunk 1]
```diff
@@ -136,8 +136,8 @@ class CircuitBreakerSpec extends AkkaSpec with BeforeAndAfter {
           Thread.sleep(500.millis.dilated.toMillis)
         }
       }
-      within(300.millis) {
-        awaitCond(breaker().currentFailureCount == 1, 100.millis.dilated)
+      within(400.millis) {
```
Comment (drewhk) [inline akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala]: Isn't this still too tight?

Comment (ktoso) [inline akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala]: It's using the 

```
  def shortCallTimeoutCb()(implicit system: ActorSystem, ec: ExecutionContext): Breaker =
    new Breaker(new CircuitBreaker(system.scheduler, 1, 50.millis.dilated, 500.millis.dilated))
```

so technically should trigger quite fast... Shall I increase to 300ms?

**Comment by ktoso** [comment]
Increased timeout a bit more

**Comment by johanandren** [comment]
LGTM

**Comment by ktoso** [comment]
PLS BUILD

**Comment by ktoso** [comment]
Want to make sure this actually helped

**Comment by ktoso** [comment]
Will want to fix somehow else

**Pull request closed without merging**